### ISSUE TITLE
Set up automatic deploying on Heroku

### ISSUE THREAD

**Issue by husainhz7** [AMU-Code-Squad/food-up#41]
Will give new contributers a quick look at the current state of the project.

**Comment by version0chiro** [comment]
I can do this with GitHub actions making a proper workflow but will need Heroku secret for it

**Comment by pnijhara** [comment]
Yeah, that can be done. Please use Heroku secrets from your side and email a copy.

**Comment by Sathvik902** [comment]
Hello!
can I work on this issue?
